modules/apps/headless/headless-delivery/headless-delivery-test/src/testIntegration/java/com/liferay/headless/delivery/resource/v1_0/test/StructuredContentResourceTest.java

```diff
@@ -330,6 +332,7 @@ public void testGetContentStructureStructuredContentsPage()
 		super.testGetContentStructureStructuredContentsPage();
 
 		_testGetContentStructureStructuredContentsPageWithFilter();
+		_testGetContentStructureStructuredContentsPageWithoutLocalizationWithFilter();
```

Comment:
@gergelyszalay, I would change the names of both of the methods, maybe something like: 

_testGetContentStructureStructuredContentsPageLocalizedWithFilter
_testGetContentStructureStructuredContentsPageUnlocalizedWithFilter